packages/manager/apps/web/client/app/private-database/dashboard/private-database.html

```diff
@@ -193,20 +193,66 @@
             ></p>
         </div>
 
-        <div data-ng-controller="PrivateDatabaseTabsCtrl">
-            <wuc-ovh-tabs
-                tabs="tabs"
-                selected-tab="selectedTab"
-                product="privateDatabase"
-                change-tab="setSelectedTab"
-                tr="tr"
-            ></wuc-ovh-tabs>
-            <div class="tab-content">
-                <div
-                    class="tab-pane active"
-                    data-ng-include="!server.isExpired && 'private-database/' + toKebabCase(selectedTab) + '/private-database-' + selectedTab.toLowerCase() + '.html' || ''"
-                ></div>
-            </div>
+        <!-- <div > -->
+        <oui-header-tabs>
+            <oui-header-tabs-item
+                href="{{ ctrl.stateLink }}"
```

Comment:
maybe one-time binding?

Reply:
Thanks @radireddy for pointing out this.

It has been added as a todo on the parent task (see: #3330).